plugins.ts

```diff
@@ -35,10 +35,13 @@ export const inject = (deps: PluginDeps) => {
         attemptP(async () => {
             if (parsedArgs.logPluginRequests)  {
                 const encoder = new TextEncoder()
-                const output = pluginRequestToString (plugin) (cmd)
+                let output = pluginRequestToString (plugin) (cmd)
                 await stdout.write(encoder.encode(`*** START Call to ${plugin.name} ***\n`))
                 await stdout.write(encoder.encode(`${output}\n`))
                 await stdout.write(encoder.encode(`*** END of call to ${plugin.name} ***\n`))
+                if (parsedArgs.debug) {
```

Comment:
Jimmy, we discussed meeting to review how to do this in a way that avoids a SoC violation.

Do you mind doing the following please:
1) Create an issue to fix the SoC violation here
2) Add a TODO comment that links to the issue.

Thanks!

Reply:
Got it! The issue is here: https://github.com/ecadlabs/taqueria/issues/732

Will push a commit to contain this TODO comment.